yaml/parse.ts

```diff
@@ -23,8 +22,10 @@ export interface ParseOptions {
    * Specifies a schema to use.
    *
    * Schema class or its name.
+   *
+   * Passing Schema class is deprecated. Use schema name instead.
    */
-  schema?: string | Schema;
```

Comment:
I changed `Schema` to `unknown` here for now. This is required to pass the doc linter check. If we keep `Schema` here, we need to document all properties of `Schema` and `Type` to pass the doc linter check, which involves huge amount of work. However we plan to hide these from public API soon #5117